src/universal/components/UserHub/UserHub.js

```diff
@@ -17,23 +19,74 @@ const UserHub = (props) => {
     dispatch(push('/me/settings'));
   };
 
+  const onCloseSettingsClick = (event) => {
+    const {dispatch} = props;
+    event.preventDefault();
+    dispatch(push('/me'));
+  };
+
+  const onSignOutClick = (event) => {
+    const {dispatch} = props;
+    event.preventDefault();
```

Comment:
there is no default event action for a click unless it's a checkbox or a submit button or somethin. will remove (if I'm wrong, lemme know!)